### PR TITLE
Run regexp system test in streaming mode.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review
@vekterli : FYI

### CODE REVIEW

File: tests/search/regexp/regexp.rb
Comment (vekterli) [inline tests/search/regexp/regexp.rb:155]: https://github.com/vespa-engine/vespa/pull/29932 should resolve cased matching for streaming. Consider holding off until that PR is merged, at which point this special case can be removed

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**
